### PR TITLE
google-java-format 1.1 (new formula)

### PR DESCRIPTION
**Pull request by dlackty**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/google-java-format.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    system "mvn", "install", "-DskipTests=true", "-Dmaven.javadoc.skip=true", "-B"
+    libexec.install "core/target/google-java-format-#{version}-all-deps.jar"
+
+    bin.write_jar_script libexec/"google-java-format-#{version}-all-deps.jar", "google-java-format"
+  end
+
+  test do
+    path = testpath/"foo.java"
+    path.write "public class Foo{\n}\n"
```
Comment (MikeMcQuaid) [inline Formula/google-java-format.rb]: `(testpath/"foo.java").write "public class Foo{\n}\n"`

File: Formula/google-java-format.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    output = `#{bin}/google-java-format #{path}`
+    assert_match output, "public class Foo {}\n"
```
Comment (MikeMcQuaid) [inline Formula/google-java-format.rb]: `assert_match "public class Foo {}", shell_output("#{bin}/google-java-format foo.java")

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by dlackty** [comment]
@MikeMcQuaid Updated and CI passed. Thanks for reviewing!

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**